### PR TITLE
correct message language selection

### LINKED ISSUE #6: get_passages(stop_name) returns only one stop
Hi,

I've tried your example.py with service.get_passages("Saint-Denis") and it returns only data for stop_id: '5719' and it's missing info about stop_id 5152.

### PR DESCRIPTION
**Pull request by danito**
Returned messages are in fr or nl for passingTimes
Closes #6

### CODE REVIEW

**Review by helldog136 — CHANGES_REQUESTED** [review]
Ok maybe by default we will put the same language for both? so let's put 'fr',' fr'  :)

**Review by helldog136 — DISMISSED** [review]
oopsie :D

**Comment by helldog136** [comment]
I'm kind of struggling with the PR (first time :D ) Please put the same language for both values for the default ('fr','fr') . For the rest of course the fix is correct :D

File: pystibmivb/service/STIBService.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ async def get_passages(self, stop_name, line_filters=None, max_passages=15, lang
         stop_infos = await self._shapefile_service.get_stop_infos(stop_name)
 
         if lang is None or lang == '':
-            lang = ('fr', 'en')
+            lang = ('fr', 'nl')
```
Comment (helldog136) [inline pystibmivb/service/STIBService.py:30]: Put ('fr','fr') as default (or even 'nl', 'nl' i don't care but it has to be consistent)

File: pystibmivb/service/STIBService.py [hunk 2]
```diff
@@ -51,7 +51,7 @@ async def get_passages(self, stop_name, line_filters=None, max_passages=15, lang
                     except KeyError:
                         pass
                     try:
-                        if message.upper() == "FIN DE SERVICE" or message.upper() == "EINDE VAN SERVICE":
+                        if message.upper() == "FIN DE SERVICE" or message.upper() == "EINDE DIENST":
```
Comment (helldog136) [inline pystibmivb/service/STIBService.py:54]: Yeah i'm sorry I'm not very good at Dutch :/

**Review by helldog136 — CHANGES_REQUESTED** [review]

File: pystibmivb/service/STIBService.py (hunk 1 above)
Comment (danito) [reply pystibmivb/service/STIBService.py:30]: I didn't understand your logic here.
We have LANG_MESSAGE = 1  by default
So when we have 
```
                    message = ""
                    try:
                        message = json_passage["message"][lang[LANG_MESSAGE]]
```
we have lang[1] ("en"), so I changed it to lang={"fr","nl"} but checking better, I see there's no way to change LANG_MESSAGE, get_passages() expects a tuple for lang and not a value for LANG_MESSAGE.
So how do you change the language for the messages?
Do you expect to do get_passages("Saint-Denis", lang={"fr","fr"})?
Why not having something like lang = "fr", and do a check if lang is in {"fr","nl"} ?

Comment (helldog136) [reply pystibmivb/service/STIBService.py:30]: Yeah it seems that messages can be in three languages (fr,nl,en) but stop names are only in two (fr,nl) 
The lang variable is a tuple containing the configuration for the lang for the stop_names (index=0) and the lang for the messages (index=1)

Do you expect to do get_passages("Saint-Denis", lang={"fr","fr"})? => yes

I haven't put a check to ensure that the lang was in 'fr','nl' or 'fr','nl','en' 

In future versions i plan on adding more input verfications and more exceptions everywhere but i wanted to keep it simple for me at the start.

**Comment by helldog136** [comment]
translation (Einde Dienst) change added in latest 1.3.0 release

We will further discuss this lang logic in an issue if you want :) please open one

**Pull request closed without merging**